docs/cloud-edge/modules/response-headers.mdx

```diff
@@ -0,0 +1,344 @@
+import HttpHeaderTemplates from "./_http_header_templates.mdx";
+
+# Response Headers
+
+---
+
+## Overview
+
+This module adds and removes headers from the HTTP response before it is
+returned to the client. This is useful for stripping internal headers or
+enforcing the use of security headers without modifying upstream applications.
+
+You may [interpolate variables into header values](#variable-interpolation) to make them dynamic.
+
+## Quickstart
+
+### Agent CLI
+
+######
+
+```bash
+ngrok http 80 \
+  --response-header-add='content-security-policy: self' \
+  --response-header-add='dial-duration: ${.backend.dial_duration}' \
+  --response-header-remove='internal-trace-id'
+```
+
+### Agent Configuration File
+
+```yaml
+tunnels:
+  example:
+    proto: http
+    addr: 80
+    response_headers:
+      add:
+        [
+          "content-security-policy: self",
```

Comment:
obviously both are valid, but i think we use the hyphen syntax for arrays in yaml in other areas of the docs:

```
    response_headers:
      add:
        - "content-security-policy: self"
        - "dial-duration: ${.backend.dial_duration}"
      remove:
        - internal-trace-id
```

Reply:
fixed